src/main/Core/BrowserWindow/BrowserWindowModule.ts

```diff
@@ -166,7 +166,7 @@ export class BrowserWindowModule {
 
         ipcMain.on("escapePressed", () => shouldHideWindowOnEscapePressed() && browserWindowToggler.hide());
 
-        app.on("second-instance", () => browserWindowToggler.toggle());
+        app.on("second-instance", () => browserWindowToggler.showAndFocus());
```

Comment:
@oliverschwendener Any reason for using `showAndFocus` instead of `toggle`? This change means that the workaround for Wayland is unable to close the window with the "hotkey"

Reply:
Because I think it's generally better behavior if launching a second instance always shows the already running instance instead of toggeling. If you're using Wayland you can use the hotkey to launch a second instance and the  escape key to close the window. If that's not enough we can still pass an argument when launching the second instance, for example `--toggle` and then choose to toggle instead of showing the already running instance.

Reply:
For example:

```typescript
app.on("second-instance", (_, argv) => {
    if (argv.includes("--toggle")) {
        browserWindowToggler.toggle();
    } else {
        browserWindowToggler.showAndFocus();
    }
});
```

Reply:
That makes sense. I've updated the wiki to reflect this